lib/CPAN/Meta/Spec.pm

```diff
@@ -1080,16 +1080,18 @@ ordinary comparison operators.  For example:
   }
 
 If the only comparison needed is whether an installed module is of a
-sufficiently high version, a direct test may be done using the string
-form of C<eval> and the C<use> function.  For example, for module C<$mod>
-and version prerequisite C<$prereq>:
+sufficiently high version, a direct test may be done using the C<VERSION>
+method.  For example, for module C<$mod> and version prerequisite
+C<$prereq>:
 
-  if ( eval "use $mod $prereq (); 1" ) {
+  use Module::Load 'load';
+  if ( $mod =~ m/\A[\w:']+\z/a and eval { load $mod; $mod->VERSION($prereq); 1 } ) {
     print "Module $mod version is OK.\n";
   }
 
-If the values of C<$mod> and C<$prereq> have not been scrubbed, however,
-this presents security implications.
+The regexp checks that C<$mod> is a module name before passing it to
```

Comment:
This is true, but the check is specifically checking for what Module::Load uses to determine whether to enable its filename behavior, so it seemed appropriate. But your wording is more precise anyway.